frontend/src/models/devices.ts

```diff
@@ -117,6 +118,20 @@ export default createModel({
           console.warn(error)
         })
     },
+    async unShare(device: IDevice) {
+      dispatch.devices.set({ destroying: true })
+      const { auth } = store.getState()
+      r3.post(`/developer/device/share/${device.id}1/${auth.user?.username}/off/false`)
```

Comment:
I'm not sure about this call. Have you tested it? There is a `1` in the url that shouldn't be there. Also I'm not sure about the `off/false` settings. This is what I found in portal:

![image](https://user-images.githubusercontent.com/864957/86169604-bf13da00-bace-11ea-8d33-e9f29f789f1c.png)

So I think you should remove the off/false and encode the email.

Reply:
I have tested it, the number `1` was because I was deleted 4 of the 5 devices with me. I didn't delete everyone. 
yes, I have used this method to delete because it was in the swagger.

Updated